script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_4_1.txt

```diff
@@ -4,7 +4,7 @@ I think back to the incident with my father, Tohno Makihisa.
 ...I don't have any memories of playing with him though.
 Makihisa was a strict father, but at the same time, he was a quiet man who didn't show his emotions.
 <Page3>
-Do not go out, do not go into the city, people are born with different statuses, protect our noble bloodline which should not be seen as the same as the others―――
+Do not go out, do not go into the city, people are born with different statuses, protect our noble bloodline which should not be viewed the same as the others―――//alt suggestion: which should not be viewed in the same way as the others.
```

Comment:
I don't think "should not be viewed the same as the others" reads right. Should be "viewed **as** the same". That or the original is fine.